### ISSUE TITLE
Potential weirdness with ITT harvest levels

### ISSUE THREAD

**Issue by squeek502** [squeek502/TiC-Tooltips#54]
From [/u/Aireon on /r/feedthebeast](https://www.reddit.com/r/feedthebeast/comments/3yczs3/custom_tic_config_harvest_level_mismatch_problem/):

> I am getting the harvest level from the tooltip in NEI. The assembled tools have correct harvest level values but hammer, excavator, and shovel heads all show values one higher than I assigned them to be. The misinformation matters only for hammers though.
> 
> I checked config options and I was able to find pickaxeBoostRequired=true but I can't for the life of me find levelingPickaxeBoost anywhere. Here are some screenshots so you can see my problem.
> 
> http://imgur.com/a/aQVY1
> 
> I also had to increment all harvest levels by 1 for every material in the ITT config file. For example, I assigned harvest level of Void Metal material to be 8 (Terrasteel) but it was rendered as 7 (Orichalcum) in-game (tools only broke blocks up to Orichalcum harvest level). Because of that, I assigned Void Metal material harvest level of 9 instead of 8, which made Void Metal tools actually have harvest level of Terrasteel. I did not alter the harvest levels themselves, only the properties of materials.

---

The OP:

> I encountered a problem after rebalancing TiC part stats - the mining levels mismatch on tools and some tool parts. For example, I assigned Vulcanite to have harvest level labeled as 9 (Atlarus). If I log into a game, the value (9) will be displayed only for shovel, excavator, and hammer heads made of Vulcanite. As for pickaxe and other tool heads made of Vulcanite, their harvest level will be displayed as 8 (Terrasteel), one harvest level lower than I want it to be. If I use any of the tool heads made of Vulcanite (including those with the correct harvest level as seen in NEI), the resulting tool will have harvest level of 8 (Terrasteel), not 9 (Atlarus).
> 
> TL;DR: Harvest levels assigned to materials in the config are lowered by 1 for any fully crafted TiC tool and some of the tool parts like pickaxe head. At the same time, NEI displays shovel, hammer head, and excavator head of any material as having the correct harvest level that is indicated in the config.
> 
> Temporary Solution: I have went through the MaterialsOverride.cfg file and bumped every material's harvest level by 1. All of the problems with incorrect mining levels are fixed except for harvest levels of shovel, excavator, and hammer heads displayed in NEI as 1 harvest level higher than I want them to be. Is there a way I can prevent this mismatch from confusing a player playing my pack, aside from telling them that the harvest level of the complete tool is the correct one?
